apps/platform/src/lists/ListService.ts

```diff
@@ -170,21 +171,45 @@ export const populateList = async (list: List, rule: Rule) => {
     const version = oldVersion + 1
     await updateList(id, { state: 'loading', version })
 
-    type UserListChunk = { user_id: number, list_id: number, version: number }
-    const query = ruleQuery(rule, list.project_id)
-    await chunk<UserListChunk>(query, 100, async (items) => {
-        if (items.length <= 0) return
+    // look up all users in list's project
+    const query = User.query().where('project_id', list.project_id)
```

Comment:
This can most likely be filtered to only be users who have any of the included events. Could help reduce total number of users that need processing